components/provider/Cargo.toml

```diff
@@ -32,8 +30,8 @@ erased-serde = "0.3"
 smallstr = { version = "0.2", features = ["serde"] }
 downcast-rs = "1.2"
 serde = { version = "1.0", features = ["derive"] }
+icu_locid_macros = { version = "0.1", path = "../locid/macros" }
```

Comment:
that's a significant increase for a foundational component. Can we keep it as optional or dev?

Reply:
Fixed by making it an optional dependency on the hello_world feature